x-pack/plugin/src/test/resources/rest-api-spec/test/api_key/10_basic.yml

```diff
@@ -41,3 +60,67 @@ setup:
   - is_true: id
   - is_true: api_key
   - is_true: expiration
+  - transform_and_set: { login_creds: "#base64EncodeCredentials(id,api_key)" }
```

Comment:
Other clients consume our rest tests so we need to add a skip section with this feature. Check out the docs at https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/main/resources/rest-api-spec/test/README.asciidoc

Reply:
I apologize for not being clear. This test needs a skip section with a `feature` field. The value would probably be `transform_and_set`. This also needs to be added to https://github.com/elastic/elasticsearch/blob/master/test/framework/src/main/java/org/elasticsearch/test/rest/yaml/Features.java as a supporter feature so we dont skip the test